Makefile

```diff
@@ -26,6 +26,8 @@ SOLIDITY_GO_FILES = $(patsubst %.sol, %.sol.go, $(SOLIDITY_FILES))
 
 CI_IMAGE="hyperledger/burrow:ci"
 
+BIN_PATH?=${HOME}/.local/bin
+
```

Comment:
Why install stuff into ~/.local/bin and what was wrong with the old scheme?

Reply:
Now we are on modules and with Go removing the necessity to even set up `GOPATH` it seems better to make people set up or care about `GOPATH` anymore

Reply:
Open to other suggestions for installing binaries, but `.local/bin` is part of this spec: https://www.freedesktop.org/software/systemd/man/file-hierarchy.html - and happens to be what I use for rootless installs.

Reply:
If GOPATH is unset, it defaults to $HOME/go

https://golang.org/cmd/go/#hdr-GOPATH_environment_variable

Reply:
go install will go to $GOPATH/bin (which defaults to $HOME/go) and make install will go $HOME/.local/bin 
That's confusing. Best not do anything surprising for users.

Reply:
Okay, I will default to `$GOPATH/bin` and set default fo GOPATH